keyboards/dinofizz/fnrow/v1/mcuconf.h

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
Please run `qmk chibios-confmigrate` on these files.

Reply:
I have updated the chconf and mcuconf files with the output of this command (this was new info for me)

Reply:
This seems to be still outstanding.

Reply:
Thank you, apologies for wasting yoru time. I had updated chconf and halconf, not mcuconf. I have pushed a commit with the change. Are there any other conf files I should generate using this tool?